CHANGELOG

```diff
@@ -1,3 +1,23 @@
+snowplow-fractribution 0.2.0 (2023-02-23)
+---------------------------------------
+## Summary
+This release adds support for Databricks and BigQuery. From now users are allowed to filter unwanted channels as well. Under the hood we added an integration testing suite and some automations with Github actions.
+
+## Features
+Add support for Databricks and BigQuery
+Add release and pages actions (#5)
+Allow user to filter unwanted channels (#3)
+Add versioning for Docker
+
+## 🚨 Breaking Changes 🚨
+Variable names have changed, please align the new ones found in the dbt_project.yml file and the ones defined in your own project's yml file that you used to overwrite the default values. Please note that
+`snowplow__path_transforms` variable is a dictionary instead of an array.
```

Comment:
We also now require dbt 1.3 I think?

Reply:
Oh good point, I missed to add that bit eventually.